script.js

```diff
@@ -47,6 +53,27 @@ function setup() {
   nav_select_div.appendChild(nav_select);
   nav_select.value = "none";
   nav_select.className = "nav_select";
+  episodes.forEach(item => {
+    let newOption = document.createElement("option");
+
+    newOption.value = item.name;
+    newOption.innerText = `$S${String(item.season).padStart(2,0)}E${String(item.number).padStart(2,0)} - ${item.name}`;
```

Comment:
You can create a function that handles SeasonCode and then you can just call that function instead of repeating that long line of code.